### PR TITLE
Log Client IP in Producer log

### PR DESCRIPTION
**Pull request by mangchiandjjoe**
jsonPayload: {
  url: "/api/xxx/probes/readiness"   
  producer_project_id: "xxx-test"   
  api_name: "1.membertarget_endpoints_xxx_test_cloud_goog"   
  api_method: "1.xxxe_Autogenerated_Unrecognized_Get_Method_Call"   
  location: "us-central1"   
  api_version: "1"   
  request_latency_in_ms: 56   
  request_size: 829   
  http_response_code: 200   
  response_size: 159   
  log_message: "Method: 1.xxx.Google_Autogenerated_Unrecognized_Get_Method_Call"   
  http_method: "GET"   
  timestamp: 1493917533.7669845,
  client_IP: "xxx.xxx.xxx.xxx" <---******* add something like this ********
 }

### CODE REVIEW

**Comment by wenchenglu** [comment]
Please hold this PR until we got approval from privacy team

**Comment by mangchiandjjoe** [comment]
Sure I will. Thanks.

File: contrib/endpoints/src/api_manager/service_control/proto.cc [hunk 1]
```diff
@@ -830,6 +830,7 @@ const char kLogFieldNameLogMessage[] = "log_message";
 const char kLogFieldNameRequestLatency[] = "request_latency_in_ms";
 const char kLogFieldNameUrl[] = "url";
 const char kLogFieldNameErrorCause[] = "error_cause";
+const char kLogFieldClientIP[] = "client_IP";
```
Comment (qiwzhang) [inline contrib/endpoints/src/api_manager/service_control/proto.cc]: should it be all lower case to be consistent?

Comment (mangchiandjjoe) [inline contrib/endpoints/src/api_manager/service_control/proto.cc]: Done

**Comment by tangiel** [comment]
Should this be client_ip, since we don't make producer_project_id producer_project_ID?

**Comment by qiwzhang** [comment]
Yes, the code has been changed to client_ip.  But PR description did not change.  Hi @mangchiandjjoe could you update the description and also fix the test.

**Comment by lizan** [comment]
cloudendpoints/esp#216

**Pull request closed without merging**